### DISCUSSION TITLE
How to set job.env use variables that have been defined in job.env . thanks

### DISCUSSION THREAD

**Discussion by shinhwagk** [community/community#26142] [Actions]
<pre><code>```yml
jobs:
  step1:
    runs-on: ubuntu-latest
    env:
  var1: hello
  var2: "${var1}_world"
```
</code></pre>

**Comment by adamrushuk** [comment, marked as answer]
<p>That is not supported right now, I asked a similar question, but there was a solution: <a href="https://github.community/t/how-can-we-concatenate-multiple-env-vars-at-workflow-and-job-level/17744/2">https://github.community/t5/GitHub-Actions/How-can-we-concatenate-multiple-env-vars-at-workflow-and-job/m-p/48521#M7203</a></p>
